pkg/addons/data/Role.rbac.authorization.k8s.io/openshift-etcd/etcd-operator.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: etcd-operator
+  namespace: openshift-etcd
+rules:
+- apiGroups:
+  - etcd.database.coreos.com
+  resources:
+  - etcdbackups
+  - etcdrestores
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - events
+  - secrets
+  verbs:
+  - "*"
```

Comment:
This is pretty wide.  Open follow-up issue to tune down later if possible.

Reply:
#378

Reply:
It was original for od based etcd. I think we should be good only with `secrets` and maybe `events`. This should be straight forward task for anybody to pick